ROADMAP.md

```diff
@@ -0,0 +1,46 @@
+# Roadmap
+
+## Towards seamless upgrades
+
+TBD
+
+## Pipeline
+
+### Support feature flags
+
+TBD
```

Comment:
Can we list some examples of things an operator might enable/disable?

- Affinity Assistant
- Custom Tasks
- Maybe disabling ClusterTasks?